mofacts/client/views/experimentReporting/studentReporting.js

```diff
@@ -1,16 +1,61 @@
+
 import {setStudentPerformance} from '../../lib/currentTestingHelpers';
 import {INVALID} from '../../../common/Definitions';
 import {meteorCallAsync} from '../..';
+import gauge, {
+  Gauge,
+  Donut,
+  BaseDonut,
+  TextRenderer
+
+} from '../../lib/gauge.js';
+import { Session } from 'meteor/session';
```

Comment:
Is this necessary? I believe this is a meteor framework global, but I may have imported it to get rid of linting errors in other files